Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -948,17 +948,27 @@ RefPtr<CSSValue> consumeAttr(CSSParserTokenRange args, const CSSParserContext& c
         return nullptr;
 
     CSSParserToken token = args.consumeIncludingWhitespace();
+
     AtomString attrName;
     if (context.isHTMLDocument)
         attrName = token.value().convertToASCIILowercaseAtom();
     else
         attrName = token.value().toAtomString();
 
-    if (!args.atEnd())
-        return nullptr;
+    if (!args.atEnd() && args.peek().type() == CommaToken) {
+        if (!consumeCommaIncludingWhitespace(args))
+            return nullptr;
```

Comment:
```suggestion
    if (!args.atEnd() && !consumeCommaIncludingWhitespace(args)) {
        return nullptr;
```

I think this should be sufficient.